config.json

```diff
@@ -667,7 +667,7 @@
         "name": "Allergies",
         "uuid": "9d33d21c-e695-427f-9f58-dd9498d61318",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["classes", "math", "numbers", "for-loops", "arrays"],
```

Comment:
Here is my solution : https://exercism.org/tracks/javascript/exercises/allergies/solutions/TomPradat

I used array transformations a lot, but looking at the community solutions, I found [this one](https://exercism.org/tracks/javascript/exercises/allergies/solutions/enricofelinto) that I like better. That's why I didn't put `array-transformations` as a prerequisite, you can easily replace the `filter` method with a for loop

`math` is not in the list of concepts but is present a lot in other exercises' prerequisites.

Reply:
Myeah I wonder what happens with `math`. I guess we'll find out after merging this :P

Reply:
I think I read somewhere that the exercise is only unlocked after all concepts have been completed. I couldn't find where I saw that though